Model/Source/AbstractInsurancePossibilities.php

```diff
@@ -4,11 +4,27 @@
 
 namespace MyParcelNL\Magento\Model\Source;
 
+use Exception;
 use Magento\Framework\Data\OptionSourceInterface;
+use MyParcelNL\Sdk\src\Factory\ConsignmentFactory;
+use MyParcelNL\Sdk\src\Model\Consignment\AbstractConsignment;
+use MyParcelNL\Sdk\src\Services\CountryCodes;
 
 abstract class AbstractInsurancePossibilities implements OptionSourceInterface
 {
 
+    protected string $type;
```

Comment:
gaat dit goed? doen we php >7.1 syntax in magento?

Reply:
voor zover ik weet gebruiken we php 8.2

Reply:
Zitten al onze Magento 2 klanten op php >= 8.2??? 😳 Dat lijkt mij niet, kun je dat ff nalopen?

Onze composer.json zegt `>= 7.0 || >= 8.0` (en er zit zelfs nog php 5 in maar dat zal wel niet kloppen)

Reply:
ik heb de types verwijderd.